src/sentry/lang/native/symbolizer.py

```diff
@@ -1,14 +1,100 @@
 try:
     from symsynd.driver import Driver
     from symsynd.report import ReportSymbolizer
+    from symsynd.macho.arch import get_cpu_name
     have_symsynd = True
 except ImportError:
     have_symsynd = False
 
+from django.db import connection
 from sentry import options
 from sentry.lang.native.dsymcache import dsymcache
 
 
+SDK_MAPPING = {
+    'iPhone OS': 'iOS',
+}
+
+
+def get_sdk_from_system_info(info):
+    if not info:
+        return None
+    try:
+        sdk_name = SDK_MAPPING[info['system_name']]
+        system_version = tuple(int(x) for x in (
+            info['system_version'] + '.0' * 3).split('.')[:3])
+    except LookupError:
+        return None
+
+    return {
+        'dsym_type': 'macho',
+        'sdk_name': sdk_name,
+        'version_major': system_version[0],
+        'version_minor': system_version[1],
+        'version_patchlevel': system_version[2],
+    }
+
+
+def find_system_symbol(img, instruction_addr, system_info=None):
```

Comment:
Given the current index, anyone any objections to these two queries? Unfortunately they are sent individually for all N frames. I could union them up with a bit of extra work but the only way to really improve this would be a stored procedure i think.